packages/bierzo-wallet/src/routes/registerName/index.stories.tsx

```diff
@@ -34,9 +36,33 @@ async function onSubmit(_: object): Promise<void> {
   action("onSubmit")();
 }
 
-async function validate(_: object): Promise<object> {
-  action("validate")();
-  return {};
+async function validate(values: object): Promise<object> {
+  const formValues = values as FormValues;
+  const errors: ValidationError = {};
+
+  const username = formValues[REGISTER_USERNAME_FIELD];
+  if (!username) {
+    errors[REGISTER_USERNAME_FIELD] = "Required";
+    return errors;
+  }
+  const checkResult = isValidIov(username);
+
+  switch (checkResult) {
+    case "not_iov":
+      errors[REGISTER_USERNAME_FIELD] = "Personalized address must include namespace suffix";
+      break;
+    case "too_short":
+      errors[REGISTER_USERNAME_FIELD] = "Personalized address should be at least 3 characters";
+      break;
+    case "too_long":
+      errors[REGISTER_USERNAME_FIELD] = "Personalized address should be maximum 64 characters";
+      break;
+    case "wrong_chars":
+      errors[REGISTER_USERNAME_FIELD] =
+        "Personalized address should contain 'abcdefghijklmnopqrstuvwxyz0123456789-_.' characters only";
+      break;
```

Comment:
Can you add an empty `case "valid"` where nothing happens and throw an error in `default:`? This ensures an error is not ignored in case of a missing error case.